spec/js/post_office_search_spec.ts

```diff
@@ -88,16 +79,16 @@ describe('Post Office Search', () => {
   it('renders post office search form fields that are enabled', () => {
     const { addressField, cityField, stateField, zipcodeField, searchButton } = getFormElements();
 
-    expect(addressField).toBeEnabled();
-    expect(cityField).toBeEnabled();
-    expect(stateField).toBeEnabled();
-    expect(zipcodeField).toBeEnabled();
+    assert.equal(addressField.disabled, false);
+    assert.equal(cityField.disabled, false);
+    assert.equal(stateField.disabled, false);
+    assert.equal(zipcodeField.disabled, false);
 
-    expect(addressField).toBeVisible();
-    expect(cityField).toBeVisible();
-    expect(stateField).toBeVisible();
-    expect(zipcodeField).toBeVisible();
-    expect(searchButton).toBeVisible();
```

Comment:
we could import `chai` and keep this syntax if we wanted to right?

Reply:
> we could import `chai` and keep this syntax if we wanted to right?

We could, yeah. I considered it briefly, but a few considerations:

- Not all of these assertions would have equivalent Chai assertions, at least out-of-the-box (e.g. `toBeVisible`). There's similar third-party Chai plugins like [`chai-dom`](https://www.npmjs.com/package/chai-dom) though.
- The way plugins work in Chai require global setup. We have that already with `spec_helper.js`, but I kinda like the purity of `node:test` and `node:assert` in being self-contained and not depending on some external state.
- I kinda find some of the helpers to be more verbose and more opaque? Like, I understand what [`HTMLInput#disabled`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/disabled) is, why do I need a more verbose, bespoke DSL to assert against it?

Though I actually overlooked that these are `toBeVisible`, not all `toBeEnabled` like I had initially read it. I'd sorta assume that Testing Library helpers will only find visible elements to begin with and therefore the assertions are redundant, but I'll double-check.

Reply:
Confirmed that Testing Library will fail if it can't find visible elements:

```diff
diff --git a/assets/js/post_office_search.tsx b/assets/js/post_office_search.tsx
index 4d7fd34e..44e542e6 100644
--- a/assets/js/post_office_search.tsx
+++ b/assets/js/post_office_search.tsx
@@ -16,3 +16,3 @@ export function render() {
   root.render(
-    <form>
+    <form style={{ display: 'none' }}>
       <FullAddressSearch
```

>  Error [TestingLibraryElementError]: Unable to find an accessible element with the role "button" and name "in_person_proofing.body.location.po_search.search_button"
>  
>  There are no accessible roles. But there might be some inaccessible roles. If you wish to access them, then set the `hidden` option to `true`. Learn more about this here: https://testing-library.com/docs/dom-testing-library/api-queries#byrole